__mocks__/child_process.js

```diff
@@ -0,0 +1,14 @@
+
+/* eslint-disable no-unused-vars */
+
+'use strict';
+
+let cp = jest.genMockFromModule('child_process');
+
+function execAsync(command, cb) {
```

Comment:
As we spoke about earlier, this function is mocked for the purpose of promisify. It may be worth it to test seperating the callback params into seperate arguments to see if it still works with the promisfication by utils.js
` cb(null, 'standard output', 'null') `
if it does indeed work, this approach would be a better mock as it is the expected shape of params that are passed into the real childprocess.exec cb:  `cb(error, stdout, stderr)`

Reply:
util.promisify only takes the first two args, so combining stdout and stderr into object